### PR TITLE
feat(popover): add component + cleanup

### PR DESCRIPTION
**Pull request by rbordeanu**
Screenshot 2019-09-10 at 17 27 27

Simple popover based on mat-menu (Material doesn't offer such widget by default)

You can configure x- and y- positions and apply a css class for the panel (to change max-width for example), and of course provide the content as a template.

### CODE REVIEW

**Comment by aifrim** [comment]
Add yourself as [code owner](https://github.com/fusionfabric/finastra-design-system/blob/feature/popover/.github/CODEOWNERS#L21)

File: apps/angular-test-app/src/app/app.component.html [hunk 1]
```diff
@@ -14,19 +14,20 @@ <h6 class="uxg-h6">DESIGN SYSTEM</h6>
         <a mat-list-item routerLink="/table" routerLinkActive="uxg-list-item-selected">
           <p>Table</p>
         </a>
+        <a mat-list-item routerLink="/popover" routerLinkActive="uxg-list-item-selected">
```
Comment (aifrim) [inline apps/angular-test-app/src/app/app.component.html]: Add your component to the main component list on the home page as well.

Comment (rbordeanu) [reply apps/angular-test-app/src/app/app.component.html]: done

**Comment by rbordeanu** [comment]
> Add yourself as [code owner](https://github.com/fusionfabric/finastra-design-system/blob/feature/popover/.github/CODEOWNERS#L21)

Done

**Comment by rbordeanu** [comment]
> Remove `themes/angular-theme/ffdc-uxg-angular-theme-0.13.1.tgz `

It's removed.

**Comment by aifrim** [comment]
@rbordeanu 
let's do the `component-name.component.scss` to `component-name.theme.scss` rename in another PR.
I will merge this one.

**Comment by aifrim** [comment]
> I'm sorry but this is not ready for merge.
> [mat-menu is already ng-template](https://github.com/angular/components/blob/master/src/material/menu/menu.html), so you're wrapping a ng-template in another ng-template then exposing same properties ([less in fact](https://material.angular.io/components/menu/api)).
> Regarding the menu-item css, it is not necessary since mat-menu accept any content, not only mat-item.
> There is no point using this component over regular mat-menu.

Then you should have done a review and asked for changes.

**Pull request merged**